core/contractHost.js

```diff
@@ -23,15 +23,7 @@ function makeContractHost(E, evaluate) {
   const seatDescriptions = makePrivateName();
   // from installation to source code string
   const installationSources = makePrivateName();
-
-  function descriptionCoercer(allegedDescription) {
-    const seatDesc = seatDescriptions.get(allegedDescription.seatIdentity);
-    mustBeSameStructure(seatDesc, allegedDescription);
```

Comment:
Yeah, I suspect we can't simply lose this check. But we might be able to refactor it elsewhere.

Does Zoe still use the `contractHost`? If not, where does this issue reappear in Zoe's seat logic?